charts/descheduler/Chart.yaml

```diff
@@ -1,7 +1,7 @@
 apiVersion: v1
 name: descheduler
-version: 0.19.0
-appVersion: 0.19.0
+version: 0.20.0
+appVersion: 0.20.0
```

Comment:
descheduler 1.20 is not out yet, do you expect the PR to be merge after 1.20 is released?

Reply:
I was hoping that this could be merged and released in version `0.20.0`.
Should I not have modified this file ?

Reply:
That's still doable. It just feels strange to merge anything with 0.20 before the descheduler image for the release is published. I was just making sure :) No blocker here.

Reply:
If you want, I can revert the file to be more in line with other pull requests. It's a quick change..

Reply:
I will defer this to @damemi and @seanmalloy

Reply:
Technically the master branch correlates to the upcoming release, so this is a change we'll have to make anyway before the release. I don't see a problem making that change now if @seanmalloy doesn't either